src/cascadia/TerminalSettingsEditor/Profiles.xaml

```diff
@@ -87,17 +87,9 @@ the MIT License. See LICENSE in the project root for license information. -->
                                         Value="100"
                                         SmallChange="1"
                                         LargeChange="10"/>
-                    <ComboBox x:Uid="Profile_ColorScheme">
-                        <ComboBox.Items>
-                            <ComboBoxItem Content="Campbell"/>
-                            <ComboBoxItem Content="Campbell Powershell"/>
-                            <ComboBoxItem Content="Vintage"/>
-                            <ComboBoxItem Content="One Half Dark"/>
-                            <ComboBoxItem Content="One Half Light"/>
-                            <ComboBoxItem Content="Tango Dark"/>
-                            <ComboBoxItem Content="Tango Light"/>
-                        </ComboBox.Items>
-                    </ComboBox>
+                    <ComboBox x:Uid="Profile_ColorScheme"
+                              ItemsSource="{x:Bind ColorSchemeList, Mode=OneWay}"

```

Comment:
this will not scale to support the combo box containing little color chips, but we can cross that bridge when we come to it